lib/templates/haml/scaffold/_form.html.haml

```diff
@@ -0,0 +1,12 @@
+-# frozen_string_literal: true
+= simple_form_for(@<%= singular_table_name %>) do |f|
+  = f.error_notification
+  = f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present?
+
+  .form-inputs
+  <%- attributes.each do |attribute| -%>
```

Comment:
__[HAML-Lint]__

Illegal nesting: nesting within plain text is illegal.


<!-- Comment by [SideCI](https://sideci.com) -->